calico_upgrade/pkg/constants/help.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package constants
+
+const (
+	DatastoreIntro = `Set the Calico datastore access information in the environment variables or
+or supply details in a config file. If you are using config files, use the v1
+format of the config file (--apiconfigv1 param), and the v3 format of the
+config (--apiconfigv3 param). If you are using environment variables, the v1
+environments are all prefixed with "APIV1_"
```

Comment:
suggest we give an example, e.g.
  "... are all prefixed with "APIV1_", e.g. "APIV1_ETCD_ENDPOINTS."